### PR TITLE
Stacksmith merger

### PR DESCRIPTION
**Pull request by sameersbn**
- uses `gcr.io/stacksmith-images/ubuntu-buildpack:14.04-r07` baseimage
- uses node harpoon module

### CODE REVIEW

**Comment by sameersbn** [comment]
Ready to merge.
/cc @prydonius

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 
-ENV BITNAMI_APP_NAME=node \
-    BITNAMI_APP_USER=bitnami \
-    BITNAMI_APP_VERSION=6.0.0-0 \
-    BITNAMI_APP_DIR=$BITNAMI_PREFIX/nodejs
+ENV BITNAMI_IMAGE_VERSION=6.1.0-r0 \
+    BITNAMI_APP_NAME=node
 
-ENV PATH=$BITNAMI_PREFIX/python/bin:$BITNAMI_APP_DIR/bin:$BITNAMI_PREFIX/common/bin:$PATH
+RUN bitnami-pkg install node-6.1.0-0 --checksum 849b47ee214fc31f03a307272ead252a2b778d8278137b955a039175cf6d571d
```
Comment (prydonius) [inline Dockerfile]: 6.2.0 is out now :)

File: README.md [hunk 2]
```diff
@@ -2,41 +2,35 @@
...
-> distributed devices.
+> Node.js is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices.
 
 [nodejs.org](https://nodejs.org/)
 
 # TLDR
 
 ```bash
-docker run -it --name node bitnami/node
+docker run -it --name -v /path/to/app:/app node bitnami/node npm start
```
Comment (prydonius) [inline README.md]: Opened this discussion in the Ruby PR, but I'm wondering if the TLDR should just run the REPL so the command can be copied and run without any changes/behaviour that will cause the container to crash?

**Comment by sameersbn** [comment]
updated with `node-6.2.0-0`

**Comment by sameersbn** [comment]
updated!
